src/routes/Profile.jsx

```diff
@@ -0,0 +1,182 @@
+// React Imports
+import React, { useContext, useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+// Inrupt Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import CheckIcon from '@mui/icons-material/Check';
+import ClearIcon from '@mui/icons-material/Clear';
+import EditIcon from '@mui/icons-material/Edit';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+// Utility Imports
+import { getBlobFromSolid } from '../utils';
+// Contexts Imports
+import { SignedInUserContext } from '../contexts';
+// Component Inputs
+import ProfileInputField from '../components/Profile/ProfileInputField';
+import ProfileImageField from '../components/Profile/ProfileImageField';
```

Comment:
don't need both

Reply:
I believe you mean to combine them into one import line? I'll create a new index.js for `/components/Profile/`

Reply:
I mistakenly read this as importing the exact same thing twice!